DnDTests/Services/UtilsServiceTest.cs

```diff
@@ -84,55 +89,139 @@ public void ValueToAttributeModifier_EqualToTen_ReturnsString()
             Assert.That(actual, Is.EqualTo(expected));
         }
 
-        [Test]
-        public void ModifyCapabilities_ReturnsCapabilityList()
+        [TestCase(CharacterAttributes.STR)]
+        [TestCase(CharacterAttributes.DEX)]
+        [TestCase(CharacterAttributes.CON)]
+        [TestCase(CharacterAttributes.INT)]
+        [TestCase(CharacterAttributes.WIS)]
+        [TestCase(CharacterAttributes.CHA)]
+        public void ModifyCapabilities_ReturnsCapabilityList(CharacterAttributes characterAttribute)
         {
             // Arrange
-            IEnumerable<Capability> capabilities = [
-                new()
+            string value = "+2";
+
+            List<Capability> expected = Enum.GetValues(typeof(CharacterAttributes))
+                .Cast<CharacterAttributes>()
+                .Select(attribute => new Capability
                 {
-                    Name = "Test1",
-                    AssociatedAttribute = CharacterAttributes.STR,
-                    Value = "",
-                },
-                new()
+                    Name = "Test_" + attribute.ToString(),
+                    AssociatedAttribute = attribute,
+                    Value = attribute == characterAttribute ? value : string.Empty
+                }).ToList();
+
+            IEnumerable<Capability> capabilities = Enum.GetValues(typeof(CharacterAttributes))
+                .Cast<CharacterAttributes>()
+                .Select(attribute => new Capability
                 {
-                    Name = "Test1",
-                    AssociatedAttribute = CharacterAttributes.DEX,
-                    Value = "",
-                }
-            ];
+                    Name = "Test_" + attribute.ToString(),
+                    AssociatedAttribute = attribute,
+                    Value = string.Empty
+                });
 
-            List<Capability> expected = [
-                new()
-                {
-                    Name = "Test1",
-                    AssociatedAttribute = CharacterAttributes.STR,
-                    Value = "+2",
-                },
-                new()
+            // Act
```

Comment:
These are redundant and provide no information, unless the test has really such a complex case that these are needed ( and in which case is an indication that refactoring is needed )

Reply:
This have been modified in later commits, I think that new ModifyCapabilities_ReturnsCapabilityList test is no longer redundant